sdk/core/core-http/src/webResource.ts

```diff
@@ -143,6 +243,7 @@ export interface WebResourceLike {
   clone(): WebResourceLike;
 }
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
I think "prefer unknown to any" is the chorus of this explicit-module-boundary-types song.

Reply:
Done.